courses/machine_learning/deepdive2/text_classification/solutions/custom_tf_hub_word_embedding.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>typo "you TB" -&gt; "your TB"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>